admin/test/test_release.py

```diff
@@ -1842,40 +1844,187 @@ def test_tag_exists_fails(self):
             self.calculate_base_branch, '0.3.0')
 
 
-class CopyTutorialVagrantBox(SynchronousTestCase):
+class PublishVagrantMetadataTests(SynchronousTestCase):
     """
-    Tests for :func:`copy_tutorial_vagrant_box`.
+    Tests for :func:`publish_vagrant_metadata`.
     """
 
-    def test_vagrant_box_copied(self):
+    def setUp(self):
+        self.target_bucket = 'clusterhq-archive'
+        self.metadata_key = 'vagrant/flocker-tutorial.json'
+
+
+    def metadata_version(self, version, box_filename, provider="virtualbox"):
```

Comment:
In general I think defaults like this are not helpful.  What this does is obscure the fact that all current users of this API are virtualbox-specific.  This makes it harder to maintain any virtualbox-related functionality in the future since it becomes harder to find all of those users.  It's also really hard to ever change the default and be confident that some user hasn't been quietly broken.  It typically ends up being easier to leave off the default and just pass "virtualbox" from all current users.  I'd recommend leaving out defaults like this in the future even in test-only APIs.  It's fine to leave this as-is for this branch, though, in the interest of getting this merged.